scripts/tutorials/gatk/tutorial_8017.wdl

```diff
@@ -0,0 +1,356 @@
+# Tutorial_8017.wdl #######################################################
+#
+# This WDL script recapitulates the main workflow outlined in Tutorial#8017 at
+# <https://software.broadinstitute.org/gatk/documentation/article?id=8017>.
+# The tutorial provides example input data including a mini-reference
+# that finishes processing in a few minutes.
+#
+###################################################################################
+# You may use any reference and any number of paired FASTQ input files.
+##	Input data requirements:
+#	- reference FASTA, FAI index and DICT dictionary
+# 	- reference ALT index for alt-aware handling
+#   - five additional reference BWA index files
+#	- simulated paired reads in paired FASTQ format per sample each. These should be
+#	  named by sample name as this sample name becomes both the RGID and RGSM.
+#	  This workflow accommodates any number of samples. Provide these in
+#	  [["A.read1.fq", "A.read2.fq"],["B.read1.fq", "B.read2.fq"],...] array format,
+#	  and be sure to preserve the ".read#.fq" suffix.
+##	Required tools:
+#	- Picard v2.5.0
+#	- GATK v3.6
+#	- BWA v0.7.15
+#	- Java JDK v1.8
+###################################################################################
+# The workflow starts with BWA indexing of a reference fasta, scatters the processing
+# of multiple paired FASTQ files from alignment to single-sample variant calling, and
+# ends with variant calling of the multiple GVCFs to produce a multisample VCF.
+#
+# The script expects simulated reads and so omits base quality score recalibration.
+# Also, the script does not accomodate interval lists passed in via the "-L" option.
```

Comment:
Is this saying that it's not possible to run the script on a subset of intervals?